ui/address/SolidityscanReport.tsx

```diff
@@ -45,6 +45,30 @@ interface Props {
   hash: string;
 }
 
+type ItemProps = {
+  item: DistributionItem;
+  vulnerabilities: SolidityscanReport['scan_report']['scan_summary']['issue_severity_distribution'];
+  vulnerabilitiesCount: number;
+}
+
+const SolidityScanReportItem = ({ item, vulnerabilities, vulnerabilitiesCount }: ItemProps) => {
+  const bgBar = useColorModeValue('blackAlpha.50', 'whiteAlpha.50');
+  const yetAnotherGrayColor = useColorModeValue('gray.400', 'gray.500');
```

Comment:
can we use `text_secondary` variable?

Reply:
no, it's the opposite

Reply:
Should we consider changing the design? The excessive use of multiple shades of gray for the text feels overwhelming.